packages/ariakit/src/tooltip/__examples__/tooltip/test.tsx

```diff
@@ -0,0 +1,9 @@
+import { getByRole, hover, render } from "ariakit-test-utils";
+import Example from ".";
+
+
+test("render", async () => {
+    const { container } = render(<Example />);
+    await hover(container);
+    expect(getByRole("tooltip")).toBeInTheDocument();
+  });
```

Comment:
hey @AlexMunoz @diegohaz , can anyone say why is this test not working

1. I'm grabbing the element
2. Hovering on it
3. Then `tooltip` should be in the document or visible or truth

This is failing

Reply:
I think the problem is that you are hovering the container instead of the tooltip element.

Reply:
yeah but the tooltip element is not visible before hovering right ?,

Reply:
Yeah! I guess you should render the anchor `as="button"` and hover over the `getByRole("button")` element in the test.